test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelList.scss

```diff
@@ -36,9 +36,9 @@
     font-size: 18px;
     color: $uicore-icons-light;
     cursor: pointer;
-    -webkit-transition: color background-color.2s ease;
-    -moz-transition: color background-color.2s ease;
-    -o-transition: color background-color.2s ease;
+    -webkit-transition: color background-color .2s ease;
+    -moz-transition: color background-color .2s ease;
+    -o-transition: color background-color .2s ease;

```

Comment:
@bsteinbk @GerardasB FWIW, I don't think you should ever need to use these vendor prefixes - I believe create-react-app will add these for you as a post-processing step.

See: https://create-react-app.dev/docs/post-processing-css/

Reply:
I thought this was the responsibility of scss and one of the main reasons to use it

Reply:
@wgoehrig thanks for letting me know. I should've removed these rules instead of fixing immediate build issue.
Most of these rules were written before we moved to CRA, we will make sure not to add prefixes explicitly anymore and leave that for the tooling.